base/system/services/rpcserver.c

```diff
@@ -5106,7 +5107,7 @@ RChangeServiceConfig2A(
 
     InfoW.dwInfoLevel = Info.dwInfoLevel;
 
-    if (InfoW.dwInfoLevel == SERVICE_CONFIG_DESCRIPTION)
+    if (Info.dwInfoLevel == SERVICE_CONFIG_DESCRIPTION)
```

Comment:
Unnecessary

Reply:
I will remove it.
NB:
I concur. Yet, I don't understand the value of (keeping) inconsistent code: it just makes things somewhat obscure to me...
See `5142 else if (Info.dwInfoLevel == SERVICE_CONFIG_FAILURE_ACTIONS)`.

Reply:
Maybe or maybe not. But it has nothing to do with this fix, so it shouldn't happen in this PR.
And since it's not worth its own PR, it probably just shouldn't change at all.